framework/src/bundle/CoreBundleContext.h

```diff
@@ -125,6 +118,13 @@ class CoreBundleContext
    */
   ServiceRegistry services;
 
+  /**
+   * A LogService for logging framework messages via
+   * a default or user-provided LogService that are intended to be
+   * visible outside of the framework.
+   */
+  std::shared_ptr<cppmicroservices::cfrimpl::CFRLogger> logger;
```

Comment:
it's worthwhile, until we fix the root cause, to leave a comment warning others that the order of member declaration generally matters here.